index.html

```diff
@@ -8,6 +8,21 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <center>
+        <section id="header">
+            <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
+           
+            <img class ="ic_menu" src="images/ic_menu.png"/>
+
+        </section>
+
+        <section id="headline">
+            <h1 class="profileName">Harshita Jain</h1>
+            
+            <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me. </p>
+            <button class="startbtn">Start collaboration</button>
```

Comment:
- [ ] Your button text was not showing. kindly fix it 👍🏽.

Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/198844620-91895a57-f2b9-4db0-8426-450d87901c6b.png)

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/198844668-e449b83a-c272-454e-868f-1f5cd8e2ae64.png)